en: update task config for 2.0 (#46)
pingcap/docs-dm: en/task-configuration-file-full.md

name: test                      # The name of the task. Should be globally unique.
task-mode: all                  # The task mode. Can be set to `full`/`incremental`/`all`.
is-sharding: true               # Whether it is a task to merge shards.

> shard-mode: "optimistic"        # The shard DDL replication mode. Can be set to ``/`pessimistic`/`optimistic`

Reviewer: ```suggestion
shard-mode: "optimistic"        # The shard DDL replication mode. Can be set to ""/`pessimistic`/`optimistic`
```

ignore-checking-items: []       # The checking items that are ignored, including `all`/`dump_privilege`/`replication_privilege`/`version`/`binlog_enable`/`binlog_format`/`binlog_row_image`/`table_schema`/`schema_of_shard_tables`/`auto_increment_ID`
meta-schema: "dm_meta"          # The downstream database that stores the `meta` information.
remove-meta: false              # Whether to remove the `meta` information (`checkpoint` and `onlineddl`) corresponding to the task name before starting the replication task.
enable-heartbeat: false         # Whether to enable the heartbeat feature.

> heartbeat-update-interval: 1    # The update interval of `heartbeat`

Reviewer: ```suggestion
heartbeat-update-interval: 1    # The interval at which the heartbeat is updated.
```

> heartbeat-report-interval: 10   # The report interval of `heartbeat`

Reviewer: Please add a period at the end of each comment.
```suggestion
heartbeat-report-interval: 10   # The interval at which the heartbeat is reported.
```

> enable-heartbeat: false         # If the heartbeat feature is enabled, the MySQL heartbeat table in the upstream is updated regularly by DM. This can be used to estimate the delay time in replication.

Reviewer: ```suggestion
enable-heartbeat: false         # If the heartbeat feature is enabled, DM regularly updates the MySQL heartbeat table in the upstream to estimate the replication delay.
```

> heartbeat-update-interval: 1    # The interval at which the MySQL heartbeat table in the upstream is updated by DM.

Reviewer: ```suggestion
heartbeat-update-interval: 1    # The interval at which DM updates the MySQL heartbeat table in the upstream.
```

> heartbeat-report-interval: 10   # The interval at which the lag (the delay time) is estimated.

Reviewer: ```suggestion
heartbeat-report-interval: 10   # The interval at which DM estimates the lag (the delay time).
```

timezone: "Asia/Shanghai"       # The timezone.

> case-sensitive: false           # Whether the schema/table is case-sensitive.

Reviewer: ```suggestion
case-sensitive: false           # Determines whether the schema/table is case-sensitive.
```

    threads: 4                       # The number of the threads Mydumper dumps from the upstream database (4 by default).
    chunk-filesize: 64               # The size of the file generated by Mydumper (64 in MB by default).
    skip-tz-utc: true                # Ignore timezone conversion for time type data (true by default).

>     extra-args: "--no-locks"         # Other arguments of Mydumper. You don't need to manually configure table-list for `extra-args`, DM automatically generates the table-list configurable items.

Reviewer: ```suggestion
    extra-args: "--no-locks"         # Other arguments of Mydumper. You do not need to manually configure table-list in `extra-args`, because table-list is automatically generated by DM.
```

# ----------- Instance configuration -----------
mysql-instances:
  -

>     source-id: "mysql-replica-01"                                      # The `source-id` in source.toml.

Reviewer: ```suggestion
    source-id: "mysql-replica-01"  # The `source-id` in source.toml.
```
